Conformity/custom-rule/custom-rule.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/local/bin/python3
+
+from genericpath import isdir
+import os
+import argparse
+import sys
+from urllib import request, response
+import requests
+import json
+import yaml
+
+configFolder = os.path.expanduser(path='~/.conformity-custom-rule')
+configFilename = 'config.yaml'
+
+workspaceFolder = os.path.expanduser(path='./workspace')
```

Comment:
Why hardcode the directory of the sample rule?

You can provide a option for the desired path or use the local folder as well.

Reply:
Users can now set their own workspace folder.

The 'hardcoded' configFolder should stay that way because it stores credentials.